zetasql/fuzzing/zetasql_expression_proto_to_string.h

```diff
@@ -0,0 +1,31 @@
+//
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#ifndef ZETASQL_FUZZING_ZETASQL_EXPRESSION_PROTO_TO_STRING_H_
+#define ZETASQL_FUZZING_ZETASQL_EXPRESSION_PROTO_TO_STRING_H_
+
+#include "zetasql/fuzzing/common.h"
+#include "zetasql/fuzzing/zetasql_expression_grammar.pb.h"
+
+using namespace zetasql_expression_grammar;
+
+namespace zetasql_fuzzer {
+
+CONV_FN(Expression, expr);
+
+} //name space zetasql_fuzzer
```

Comment:
nit:  Two spaces before // and one space after and namespace, not name space

}  // namespace zetasql_fuzzer